### DISCUSSION TITLE
Ability to get a direct link for media files

### DISCUSSION THREAD

**Discussion by Perni1984** [tinacms/tinacms#2665] [Ideas]
We are using the media manager to also upload pdf files (e.g. for job postings). Currently there is no way in the media manager to get a direct link to the uploaded file, which breaks the UX for content editors, that just want to add a new link somewhere on the page to a specific media file.

A first resolution to the problem would be to have a "copy direct link to media file into clipboard" button.

![image](https://user-images.githubusercontent.com/9039956/157385901-85a65363-079a-420c-9e9f-7d05276753e0.png)

Any other ideas welcome.

**Comment by jamespohalloran** [comment]
This has been added in the latest release! https://tina.io/blog/tina-v-1.3.2
